### ISSUE TITLE
PT-E550W always cuts the label

### ISSUE THREAD

**Issue by Bart97** [inventree/inventree-brother-plugin#44]
Hello. I've noticed that it's when using this plugin with the E550W the label is always cut at the end, it doesn't matter if I disable the option in the settings. I'd like to have the pre-cut done between labels, but feed&cut manually when I'm done printing all the labels.

**Comment by matmair** [comment]
Have you selected the right printer model? The protocol is a bit different between the models.

**Comment by Bart97** [comment]
Yes, seems that the problem lies in the brother_ql library. This model requires qlr.no_chain_printing = False to disable cutting at the end of print, however it is never set in conversion.py. I can prepare a PR tomorrow, but I'm not sure what's the intended behavior of that single UI option, as there's half-cut and end cut that can be toggled separately. Personally I need to disable end cut only, so I can print multiple labels without wasting tape.

**Comment by matmair** [comment]
we use brother_ql-inventree, a downstream fork I maintain

**Comment by Bart97** [comment]
> we use brother_ql-inventree, a downstream fork I maintain

Got it, I'm testing a fix for this right now

**Comment by Bart97** [comment]
I've submitted a pull request that will resolve this issue https://github.com/matmair/brother_ql-inventree/pull/68

**Comment by Bart97** [comment]
Hi, @matmair what's the status on this? What would be the proper process to get the fix merged into your fork?

**Comment by Bart97** [comment]
Hello @matmair any updates on merging the fix for this issue? I'd love to see a fix in the official repository, as right now I have to manually patch the library.

**Comment by matmair** [comment]
@Bart97 I am on the road and are limited with my testing/hardware setups for a few more days, will review after that.

**Comment by Bart97** [comment]
Hi @matmair, sorry for bothering you again, any chance for merging this?

**Comment by Bart97** [comment]
Hi @matmair @SchrodingersGat is there any chance to get this fix finally merged?
